src/Authoring/TestDiagnostics/ArrayParameterTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Runtime.InteropServices;
+
+namespace TestDiagnostics
+{
+    /* TODO: 
+     * what happens if you put another random attribute on an array param? 
+     * check in WRC3 project ...
+    */
+    public class ReadOnlyArray : System.Attribute

```

Comment:
FYI, You can see the attribute I have defined in my PR.

Reply:
Yeah I saw that yesterday 👍